config/routes.rb

```diff
@@ -30,6 +31,9 @@
         get '/random', to: 'random_merchants#show'
         get '/:id/items', to: 'merchants_items#index'
         get '/:id/invoices', to: 'merchants_invoices#index'
+        get '/:id/favorite_customer', to: 'favorite_customer#show'
```

Comment:
Would prefer to see these routes (where you're using a wildcard like `:id`) in a nested resource instead of a namespace (then you won't have to hand-roll your own wildcards).